workers/src/main/java/edu/cornell/testenv/testrunner/JUnit5TestRunner.java

```diff
@@ -21,48 +30,45 @@ public class JUnit5TestRunner implements TestRunner {
      * @return boolean of whether all JUnit tests passed
      */
     @Override
-    public boolean runTest(TestEnvContext<String> context) {
+    public boolean runTest(TestEnvContext<String> context, TestOutputStream outputStream, File rootDir) {
         try {
+            //TODO: Assumption is class files live in /build/classes/java/ likely needing a resolve
+            String baseRootPath = rootDir.toString() + "/build/classes/java/".replaceAll("/", File.separator);
 
```

Comment:
Need to resolve best way to deal with this as it is not build-system independent and relies on the assumption that classes are compiled to root directory -> build -> classes -> java

Reply:
Maybe this can be added to the configuration file? We already have "test directories" as part of the config. I think that config value should be used for this

Reply:
I suppose later we can build it to take in a list of directories in case builds are across different specified directories. Or we can do it now.

Reply:
Do you want to add a parameter to the method with a list of build directories now? Might as well get ahead of it for now

Reply:
Sure